docker-compose.yml

```diff
@@ -1,4 +1,4 @@
-version: '3.7'
+version: "3.7"
```

Comment:
This are unrelated changes. Can we revert them?

Reply:
@iperdomo I think thats the linter

Reply:
Sorry!